### PR TITLE
MySQL container

### PR DESCRIPTION
**Pull request by leonhooijer**
Address #1

### CODE REVIEW

**Comment by crramirez** [comment]
Hello,

First of all I want to say thank you for your contribution to the project and the time you took to make this pull request and associate it with the reported issue.

I want to make comments by commits:

1. **Update README.md**. I agree that it is more organized if you reference the file from the read me. But the vast majority users of this repository, reach it via the docker hub page, not the github page. So this text will force them to look at right for the GitHub link (in mobile version the link won't even appear) and then look for the file. Instead of just copying and pasting from the page. A text could be added talking about `docker-compose.yml` is also in the git repository and can be extended in the way you describe.

2. **Add MySQL container to Docker Compose**. I think the beauty of this container and what the people like is its simplicity. Just one `docker run` and the system is running. So I think this is better to add a modified docker-compose.yml with external MySQL container support than modifying the default one.

Right now I am very busy, so I will take some time to accept and modify the changes. This way we can fulfill both needs of default options and advanced ones.

Regards

**Comment by leonhooijer** [comment]
Hello @crramirez,

Thank you for your reply. This still needs some additional work. I noticed the containers don't link correctly. I will keep your comments in mind when working on this PR.

I expect to continue to work on this after the weekend.

Regards

File: docker-compose.yml [hunk 1]
```diff
@@ -1,10 +1,15 @@
 version: '2'
 services:
+  mysql:
+    image: mysql:latest
+    volumes:
+      - mysql:/var/lib/mysql
   limesurvey:
+    depends_on:
+      - mysql
```
Comment (rclayton-the-terrible) [inline docker-compose.yml]: I do not believe `depends_on` will establish a `link` between containers (it's merely for controlling the lifecycle of the container in relation to other containers).

You should amend this to:

```
# ...
limesurvey:
  depends_on:
     - mysql
  links:
     - mysql
  ports: 
     - "80:80"
# ...
```

When linked, Limesurvey will need to use "mysql" as the host name:   mysql://mysql:3306

**Comment by leonhooijer** [comment]
@crramirez @rclayton-the-terrible I pushed some commits to process your comments. 

I would appreciate it if you could take a look and let me know what you think 🙂!

**Comment by rclayton-the-terrible** [comment]
@leonhooijer I'll take a look today.  Thanks for the hard work.

**Pull request merged**